icann-rdap-client/src/test_files/nic.xyz.json

```diff
@@ -0,0 +1,347 @@
+{
+  "rdapConformance": [
+    "icann_rdap_response_profile_0",
+    "icann_rdap_technical_implementation_guide_0",
+    "rdap_level_0"
+  ],
+  "notices": [
+    {
+      "title": "Status Codes",
+      "description": [
+        "For more information on domain status codes, please visit https://icann.org/epp"
+      ],
+      "links": [
+        {
+          "href": "https://icann.org/epp",
+          "title": "More information on domain status codes"
+        }
+      ]
+    },
+    {
+      "title": "Terms of Use",
+      "description": [
+        "For more information on Whois status codes, please visit https://icann.org/epp",
+        "",
+        ">>> IMPORTANT INFORMATION ABOUT THE DEPLOYMENT OF RDAP: please visit",
+        "https://www.centralnic.com/support/rdap <<<",
+        "",
+        "The Whois and RDAP services are provided by CentralNic, and contain",
+        "information pertaining to Internet domain names registered by our",
+        "our customers. By using this service you are agreeing (1) not to use any",
+        "information presented here for any purpose other than determining",
+        "ownership of domain names, (2) not to store or reproduce this data in",
```

Comment:
This is gonna be problematic. Can you change this test file where the data isn't from centralnic but has the same structure?

Reply:
The JSON files are for the tests, which I just finished. It looks like we can't use them so I rip them out of there.